config/environment/ui-test.php

```diff
@@ -100,7 +100,7 @@ public function write(string $key, string $content): void
 
             if (!empty($port)) {
                 // remove the port from URLs if any so UI tests won't fail if the port isn't 80
-                $result = str_replace($host . ':' . $port, $host, $result);
+                $result = str_replace($host . ':' . $port, $host, $result ?? '');
```

Comment:
Fixes a warning as sometimes in UI tests the `$result` variable is empty. We have null-coalesce below already so I guess this is safe.